crates/server-primitives/src/jsonrpc.rs

```diff
@@ -123,7 +123,7 @@ pub enum QueryError {
 #[derive(Debug, Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct MutateRequest {
-    pub application_id: calimero_primitives::application::ApplicationId,
+    pub context_id: calimero_primitives::context::ContextId,
```

Comment:
Just for visibility; only-peers-client will be updated [here](https://github.com/calimero-network/only-peers-client/blob/master/app/src/api/dataSource/ClientApiDataSource.ts#L39)